explore/src/main/scala/explore/tabs/ObsTabContents.scala

```diff
@@ -99,9 +99,7 @@ object ObsTabContents {
                   margin = (5, 5),
                   containerPadding = (5, 5),
                   rowHeight = 30,
-                  draggableHandle = ".tileTitle",
-                  useCSSTransforms =
```

Comment:
Does `react-grid-layout` 1.0 fix the problem? It happened when initializing the app on a multi-tile page directly, like with the tab `Observations`.

Reply:
I tested it and didn't see any flicker but could test a bit more. Maybe I can deploy a test app to check without the reusability indicators

Reply:
I tested locally and it's working. Thank you!